api/src/main/java/org/apache/iceberg/Schema.java

```diff
@@ -204,6 +204,10 @@ public Schema(int schemaId, NestedField... columns) {
     this(schemaId, Arrays.asList(columns));
   }
 
+  public Schema(Map<String, Integer> aliases, NestedField... columns) {
+    this(Arrays.asList(columns), aliases);
+  }
```

Comment:
I'd be hesitant to add to the API. This is only used for a test; can't you use an existing constructor?

Reply:
I removed it in the latest version.